ngx-fudis/projects/ngx-fudis/src/lib/components/form/localized-text-group/localized-text-group.component.html

```diff
@@ -80,6 +80,7 @@
   </div>
   <fudis-select
     *ngIf="_languageLabel | async as label"
+    #selectRef
     class="fudis-localized-text-group__select"
     [label]="label"
     [selectionClearButton]="false"
```

Comment:
Eventhouth now select is disabled when all controls have disabled state. Still user is able to click the select chevron . Nothing is happening but the chevron rotates.